HackAtUCI/irvinehacks-site
# feat: Portal UI

Summary
- Updated the UI for the portal page according to the Figma

Test Plan
- Go to ApplicantPortal.tsx and comment out lines 24-28
- Add "let status = identity.status; status = PortalStatus.pending;" 
- Test each of the possible PortalStatus states by changing what status is set to; e.g. status = PortalStatus.waived;...
- Ensure that the UI looks right

Closes #398
